docs/sources/reference/builder.md

```diff
@@ -46,6 +46,8 @@ the files needed for building that Dockerfile to that directory. To further spee
 build, you can exclude files and directories by adding a `.dockerignore` file to the same
 directory.
 
+The `.dockerignore` file also supports exclusion patterns (as per `.gitignore` files).
```

Comment:
Combine this with the paragraph above. Also, are the patterns the same or similar? If they are the same we should say so. If they are similar, we should document the differences or point to where they exist. I'm assuming they are the same. So,

---> https://gist.github.com/moxiegirl/2a8f455f0e7836377457

In most cases, it's best to put each Dockerfile in an empty directory. Then,
only add the files needed for building the Dockerfile to the directory. To
increase the build's performance, you can exclude files and directories by
adding a `.dockerignore` file to the directory. This file supports the same exclusion patterns as `.gitignore` files.